src/qml/FeatureForm.qml

```diff
@@ -455,7 +455,7 @@ Page {
                 target: attributeEditorLoader.item
 
                 function onValueChanged(value, isNull) {
-                  if( AttributeValue != value && !( AttributeValue === undefined && isNull ) ) //do not compare AttributeValue and value with strict comparison operators
+                  if( AttributeValue !== value && !( AttributeValue === undefined && isNull ) ) //do not compare AttributeValue and value with strict comparison operators
```

Comment:
Are you sure the strict comparison here is needed? If so, change the comment.

I think I am the author of the comment, but stupid me didn't put explanation (if I had one) why we need the less strict comparrison. What I can suggest is to check with numeric values or widgets working with feature ids.

Reply:
@suricactus , yeah, without the strict comparison, going from 0 to null doesn't work as the QML engine says 0 == null .

Reply:
@suricactus , that's because this condition 0 != '' returns false under QML.

Reply:
Yup, I am mostly afraid of the numeric and string comparison which works for `==` but not for `===`:
```
0 == '0' == '' == null == undefined == false
```
Let's hope for the best we don't use any the automatic type conversions from the table here:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness#loose_equality_using

What might be better is to explicitly check for `null`?

Either way, the comment needs to be fixed.